2026 Messaging Milestones (#280)
logos-co/roadmap: content/messaging/furps/core/messaging_sdk.md

---

> title: Messaging SDK

Reviewer: Messaging term is too generic
```suggestion
title: Logos-messaging SDK, a.k.a., LM SDK
```

Author: It's generic on purpose, this is as per Jarrad's directive: Messaging, Storage and Blockchain.

Author: I'll let @kaiserd handle this

date: 2025-11-13
tags:
    - messaging-furps
---
## Functionality

> 1. Setup, start and stop a messaging node.

Reviewer: ```suggestion
1. Setup, start and stop an lm node.
```
